fastlane_core/spec/helper_spec.rb

```diff
@@ -147,6 +147,11 @@
         expect(FastlaneCore::Helper.transporter_path).to match(%r{/Applications/Xcode.*.app/Contents/Applications/Application Loader.app/Contents/itms/bin/iTMSTransporter|/Applications/Xcode.*.app/Contents/SharedFrameworks/ContentDeliveryServices.framework/Versions/A/itms/bin/iTMSTransporter})
       end
 
+      it "#path_name_itmsp", requires_xcode: true do
+        value = FastlaneCore::Helper.path_name_itmsp
+        expect(FastlaneCore::Helper.path_name_itmsp).to match(value)
```

Comment:
This expectation is kinda redundant isn't it? You're checking the value with itself, which will be always true.

Reply:
Hello @minuscorp thank you so much your comment!

Is expertly this, because when this value is called in the first time it generate one value and when it called again, this value must need de same, because the name path can't change 😁

When the _path_name_itmsp_ is set in _value_ is expect that is the same in the match.

Again, thank you so much for your comment!